internal/clientcache/internal/daemon/status_handler.go

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package daemon
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"time"
+
+	"github.com/hashicorp/boundary/internal/clientcache/internal/cache"
+	"github.com/hashicorp/boundary/internal/errors"
+	"github.com/hashicorp/boundary/internal/util"
+)
+
+// RefreshTokenStatus is the status of a resource token
+type RefreshTokenStatus struct {
+	Age      time.Duration
+	LastUsed time.Duration
+}
+
+type ErrorStatus struct {
+	Error        string
+	LastReturned time.Duration
+}
+
+// ResourceStatus contains the status of a specific resource type contained in
+// the cache for a specific user.
+type ResourceStatus struct {
+	Name         string
```

Comment:
Please add `json` tags to the various structs here and anywhere else we are marshaling as the default JSON output does not match our standard casing.